packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -303,12 +303,22 @@ class TransitLegBody extends Component<Props, State> {
                 {!shouldOnlyShowAlertsExpanded && (
                   <S.CaretToggle expanded={alertsExpanded} />
                 )}
+                <S.InvisibleAdditionalDetails>
+                  <FormattedMessage
+                    defaultMessage={
+                      defaultMessages["otpUi.TransitLegBody.expandDetails"]
+                    }
+                    description="Screen reader text added to expand steps"
+                    id="otpUi.TransitLegBody.expandDetails"
+                  />
+                </S.InvisibleAdditionalDetails>
```

Comment:
The "expand" hint should be inside the conditional block above, so it is shown only if the toggle is also shown.

Reply:
Good point, I also changed this toggle to a `div` in cases where the list cannot be collapsed.